jrnl/install.py

```diff
@@ -113,7 +111,9 @@ def load_or_install_jrnl():
             print("Exiting.", file=sys.stderr)
             sys.exit(1)
 
-        upgrade_config(config)
+        if config.get("version") and config.get("version") < __version__:
```

Comment:
Why the change in behavior here?

Reply:
Optimisation, without this check it would try to upgrade the config file on every invocation